### PR TITLE
[pravila00-enumflag] Add new port

### PR DESCRIPTION
**Pull request by Pravila00**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [ ] The name of the port matches an existing name for this component on https://repology.org/ if possible, and/or is strongly associated with that component on search engines.
- [ ] Optional dependencies are resolved in exactly one way. For example, if the component is built with CMake, all `find_package` calls are REQUIRED, are satisfied by `vcpkg.json`'s declared dependencies, or disabled with [CMAKE_DISABLE_FIND_PACKAGE_Xxx](https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html).
- [x] The versioning scheme in `vcpkg.json` matches what upstream says.
- [x] The license declaration in `vcpkg.json` matches what upstream says.
- [x] The installed as the "copyright" file matches what upstream says.
- [x] The source code of the component installed comes from an authoritative source.
- [x] The generated "usage text" is accurate. See [adding-usage](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/examples/adding-usage.md) for context.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is in the new port's versions file.
- [ ] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by dg0yt** [comment]
Already ported as `pravila00-enum-string`.

**Comment by Cheney-W** [comment]
Yes, this port should be named as `pravila00-enumflag`.

**Comment by Pravila00** [comment]
Thanks for the suggestion @Cheney-W @dg0yt. I've just renamed the port to `pravila00-enum-flag` as some other ports that I have submitted.

Depending on the reviewer, I have faced that sometimes they suggest to call my port `pravila00-*`, and sometimes they ask me to remove it. Is there any specific rule for this? Just to make the next review faster for avoiding this. Thanks!

**Comment by Cheney-W** [comment]
When reviewing a new port, we generally follow below principle for the port's name:

Ports sourced from GitHub typically use the project name.
We then search if there's a matching result in the https://repology.org/ . If found, we check if the name in the https://repology.org/ matches the source files of this port. If they match, we use that name directly. 
If not, we prepend a prefix to the name. For ports like the one you added from GitHub, the prefix would be the organization name, which is "pravila00-". 
Also, the repository name should remain the same as the original project name, without changing it to "enum-flag".

**Comment by Pravila00** [comment]
Hi @Cheney-W 

Thanks for the explanation! Now the port is called `pravila00-enumflag` which matches the repo name `enumflag` and my organization `pravila00`.

File: ports/pravila00-enumflag/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "pravila00-enumflag",
+  "version-date": "2023-04-12",
```
Comment (Cheney-W) [inline ports/pravila00-enumflag/vcpkg.json:3]: 2023?

Comment (Pravila00) [reply ports/pravila00-enumflag/vcpkg.json]: Fixed to 2024

**Review by Cheney-W — APPROVED** [review]

**Pull request merged**